### ISSUE TITLE
Loss function (MSE or RMSE) & the scale of the loss

### ISSUE THREAD

**Issue by leolya** [Sanyuan-Chen/CSS_with_Conformer#10]
When training the conformer, did you use PIT MSE or RMSE? 

To compute the MSE is it correct to use the `nn.MSELoss` in pytorch. By default it divides the loss by the total number of elements. Should I set the `reduction` to sum?

I think the scale of the loss may influence the training process (https://stats.stackexchange.com/questions/346299/whats-the-effect-of-scaling-a-loss-function-in-deep-learning), so could you please provide details of how MSE is computed?

Looking forward to your reply!
Thanks for your help @Sanyuan-Chen

**Comment by whisper0055** [comment]
Hi, I have the same problem, did you solve it ?
